pillar/elife.sls

```diff
@@ -233,6 +233,15 @@ elife:
             #profiles:  
             #    folder: /srv/profiles
 
+    multiservice:
```

Comment:
should this top key include `systemd` as a keyword, since this solution is specific to it? Or is it intended as being hidden from the user?

Reply:
I don't mind, my imagination was failing me for a while. I think 'multiservice' is short and sweet but it's totally made up. Systemd terminology would 'systemd unit group target' or something equally terse.